{{cookiecutter.project_folder}}/.gitignore

```diff
@@ -4,15 +4,15 @@ __pycache__/
 *$py.class
 
 # C extensions
-*.so
+# *.so
 
 # Distribution / packaging
 .Python
 env/
 build/
 develop-eggs/
 dist/
-downloads/
+# downloads/
```

Comment:
why is this commented out?

Reply:
to be honest I also don't know ... I basically copied over the latest `.gitignore` from compas itself. Just checked this line was modified by you 5 years ago.